Userland/Libraries/LibJS/Runtime/TypedArray.cpp

```diff
@@ -81,8 +86,12 @@ static void initialize_typed_array_from_typed_array(GlobalObject& global_object,
     if (vm.exception())
         return;
 
-    // FIXME: 4. If IsDetachedBuffer(src_data) is true, throw a TypeError exception.
-    VERIFY(src_array.viewed_array_buffer());
+    auto* source_array_buffer = src_array.viewed_array_buffer();
+    VERIFY(source_array_buffer);
+    if (source_array_buffer->is_detached()) {
+        vm.throw_exception<RangeError>(global_object, ErrorType::DetachedArrayBuffer);
```

Comment:
Same here, that comment looks like it's straight from the spec

Reply:
fixed